### PR TITLE
Modelling 3D Brain MRI using a 2D VAE

### PR DESCRIPTION
**Pull request by nathanpainchaud**

### CODE REVIEW

File: article/_posts/2021-07-02-Modelling3dBrainMriUsing2dVae.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Results
+
+![](/article/images/Modelling3dBrainMriUsing2dVae/figure2.jpg)
+![](/article/images/Modelling3dBrainMriUsing2dVae/table1.jpg)
+![](/article/images/Modelling3dBrainMriUsing2dVae/figure4.jpg)
+
+
+# Remarks
+- The authors often mention the "slice dimension", but their VAE has not explicit notion of a "slice dimension". So I
+consider this reliance on a there being a "slice dimension" to be optimist;
```
Comment (jhlegarreta) [inline article/_posts/2021-07-02-Modelling3dBrainMriUsing2dVae.md:61]: Not sure if the second sentence is complete or whether the message is clear.

**Review by jhlegarreta — COMMENTED** [review]
An inline comment.

File: article/_posts/2021-07-02-Modelling3dBrainMriUsing2dVae.md (hunk 1 above)
Comment (nathanpainchaud) [reply article/_posts/2021-07-02-Modelling3dBrainMriUsing2dVae.md]: Sorry it took me this long to get around to answering your comment. I rephrased that explanation, and I hope its easier to understand now.

Comment (jhlegarreta) [reply article/_posts/2021-07-02-Modelling3dBrainMriUsing2dVae.md]: :+1:

**Review by jhlegarreta — APPROVED** [review]

**Comment by lemairecarl** [comment]
About fig 1: Is there one latent vector per column per slice ? That's what the figure seems to suggest, but it is not said explicitly.

About fig 2: Comparing the real image with the output of "ours 256", the model seems to output a very similar slice for the 4 different volumes. There are some variations, but they don't even seem to correlate with the specifics of the real volumes. E.g. in the 3rd example, the real image has the darkest/biggest center, but in the generated slice, it's not particularly dark/big.

Otherwise, thanks for the review!

**Review by lemairecarl — APPROVED** [review]

**Comment by nathanpainchaud** [comment]
> About fig 1: Is there one latent vector per column per slice ? That's what the figure seems to suggest, but it is not said explicitly.

I'm not sure what you mean by _per column_. There's a _L_-dimensional latent vector by 2D slice.

> About fig 2: Comparing the real image with the output of "ours 256", the model seems to output a very similar slice for the 4 different volumes. There are some variations, but they don't even seem to correlate with the specifics of the real volumes. E.g. in the 3rd example, the real image has the darkest/biggest center, but in the generated slice, it's not particularly dark/big.

In that figure, I don't think you're supposed to compare the rows of generated images to the "real" row. I think the images are not encoded-decoded, but they're supposed to be generated purely from the model's prior. That's why they're comparing themselves to a GAN. At least that's why I understood.

But on a more general note, I also have quite a lot of critiques for the paper. I didn't review it because I thought it was good :stuck_out_tongue: It was mostly because it was recommended to me as a (rare) example of explicitly trying to tackle autoencoders for more than 2D data. And once I read the article, I wanted to review to get the opinion of the other lab members more knowledgeable in brain imaging, see if they thought some of the assumptions/shortcuts hold up, or if the method is indeed as badly designed as it appears.

> Otherwise, thanks for the review!

Thank you :slightly_smiling_face:

**Pull request merged**